packages/workers-shared/package.json

```diff
@@ -31,6 +31,8 @@
 		"check:type": "pnpm run check:type:tests && tsc",
 		"check:type:tests": "tsc -p ./asset-worker/tests/tsconfig.json",
 		"clean": "rimraf dist",
+		"deploy": "pnpm run deploy:asset-worker",
+		"deploy:asset-worker": "CLOUDFLARE_API_TOKEN=$WORKERS_NEW_CLOUDFLARE_API_TOKEN wrangler versions upload --experimental-versions -c asset-server-worker/wrangler.toml",
```

Comment:
I don't think `assets-server-worker` is the right path--isn't it just `asset-worker`?

Reply:
ah, such a good catch! I totes missed that old reference